sdk/tables/azure-data-tables/samples/async_samples/sample_insert_delete_entities_async.py

```diff
@@ -70,19 +70,19 @@ async def create_entity(self):
 
         table_client = TableClient.from_connection_string(self.connection_string, self.table_name)
         # Create a table in case it does not already exist
-        # [START create_entity]
         async with table_client:
             try:
                 await table_client.create_table()
             except HttpResponseError:
                 print("Table already exists")
 
+            # [START create_entity]
```

Comment:
What was the reason for this change?

Reply:
To make this async code snippet consistent with the sync one.